rebirthdb/ast.py

```diff
@@ -28,45 +28,44 @@
 from rebirthdb import ql2_pb2
 from rebirthdb.errors import QueryPrinter, ReqlDriverCompileError, ReqlDriverError, T
 
-pTerm = ql2_pb2.Term.TermType
+P_TERM = ql2_pb2.Term.TermType
 
 try:
     unicode
 except NameError:
     unicode = str
+
 try:
     xrange
 except NameError:
     xrange = range
-try:
-    {}.iteritems
 
-    def dict_items(d): return d.iteritems()
-except AttributeError:
-    def dict_items(d): return d.items()
+
+def dict_items(dictionary):
+    return dictionary.items()
```

Comment:
This will downgrade the experience on Python2.7. If that is acceptable I would like to see the function removed and make the sites more readable.

Reply:
I can not see how it would downgrade the experience.

```
$ python
Python 2.7.15 (default, Jun 17 2018, 12:46:58)

>>> {'a': 1, 'b': 2}.items()
[('a', 1), ('b', 2)]
>>> for i in {'a': 1, 'b': 2}.iteritems():
...   print i
...
('a', 1)
('b', 2)
```

Reply:
In Python2.7 the `items` method will do all the work upfront before the loop starts, and is a duplication operation. `iteritems` will only pass references around.

Reply:
Although it is a generator which returned, this should not be a problem in our case due to we are only iterating through the returned items.

In this case it should not matter, though we could convert the return value to a list and in this case both in 2.7 and 3+ would act the same. What do you think?

Reply:
I've gone back over my Python3 PR and related work that introduced this. I haven't found the issue I was expecting linked there so using `list` in all cases seems sensible. This function is duplicated in several files. So, it could be consolidated or the change duplicated.

Reply:
So, I changed it and adding list conversion. I think to extract this function to a helper file could happen in a separate PR for the same reason as extending logging